### PR TITLE
Doc: add docstrings to Brace

### PR DESCRIPTION
**Pull request by biinnnggggg**
## Overview: What does this pull request change?

Add docstrings to methods in Brace so that sphinx will create hyperlinks to these methods.

## Further Information and Comments

Related to #3597

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by JasonGrace2282** [comment]
Is it possible to add some doc tests as well?
You can see an example [here](https://github.com/ManimCommunity/manim/blob/main/manim%2Fmobject%2Fmobject.py#L404-L408).
Thanks for improving the documentation :)

File: manim/mobject/svg/brace.py [hunk 1]
```diff
@@ -126,6 +126,19 @@ def __init__(
             mob.rotate(angle, about_point=ORIGIN)
 
     def put_at_tip(self, mob, use_next_to=True, **kwargs):
+        """Puts the given mobject at the brace tip.
+
+        Parameters
+        ----------
+        mob : :class:`~.Mobject`
```
Comment (behackl) [inline manim/mobject/svg/brace.py:133]: Type hints should be added to the parameters themselves, not to the docstring.

```suggestion
        mob
```

**Review by behackl — CHANGES_REQUESTED** [review]
Looks fine, thanks! I've left one minor comment about the added type hint, please take a look.

**Review by JasonGrace2282 — APPROVED** [review]
Thanks for your work here!

**Review by behackl — APPROVED** [review]
Sorry for the long wait, thanks for your efforts!

**Pull request merged**